crates/meilisearch/src/routes/indexes/mod.rs

```diff
@@ -125,7 +129,7 @@ pub struct ListIndexes {
 }
 
 impl ListIndexes {
-    fn as_pagination(self) -> Pagination {
+    fn into_pagination(self) -> Pagination {
```

Comment:
There is no need to take ownership. According to my interpretation of [Rust's naming conventions](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv), it should be `to_pagination(&self)`